Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -642,6 +642,12 @@ Describe "Copy-GcsObject" {
             Read-GcsObject $bucket "test-dest2" | Should Be "test2-$r"
         }
 
+        It "Should overwrite existing object" {
+            $sourceObj = Get-GcsObject $bucket "test-source"
+            $sourceObj | Copy-GcsObject $bucket "test-dest2" -Force
```

Comment:
I can not. I don't throw an error. Instead I request confirmation from the user. However, the pester story for testing "ShouldConfirm" does not yet exist.